release_notes/ocp_3_11_release_notes.adoc

```diff
@@ -2582,3 +2582,107 @@ To upgrade an existing {product-title} 3.10 or 3.11 cluster to this latest
 release, use the automated upgrade playbook. See
 xref:../upgrading/automated_upgrades.adoc#install-config-upgrading-automated-upgrades[Performing
 Automated In-place Cluster Upgrades] for instructions.
+
+[[ocp-3-11-129]]
+=== RHBA-2019:1753 - {product-title} 3.11.129 Bug Fix Update
+
+Issued: 2019-07-23
+
+{product-title} release 3.11.129 is now available. The list of packages and
+bug fixes included in the update are documented in the
+link:https://access.redhat.com/errata/RHBA-2019:1753[RHBA-2019:1753] advisory.
+The container images included in the update are provided by the
+link:https://access.redhat.com/errata/RHBA-2019:1754[RHBA-2019:1754] advisory.
```

Comment:
This link 404s. Just wanted to make sure that was expected.

Reply:
Unfortunately, that's expected for now - only one of the advisories is live. There is an issue pushing certain advisories compared to others. The only reason I go ahead and merge when only one is available is because I've received complaints from support and elsewhere. It's an odd predicament that's been occurring for ~10 days and counting!